packages/itinerary-body/src/ItineraryBody/accessibility-rating.tsx

```diff
@@ -75,15 +95,28 @@ const AccessibilityRating = ({
   // External configuration may report "0.0" as 0, so include fallback
   const mapped = mapping[mappedKey] || mapping[0.0];
 
+  const accessibilityPreface = intl.formatMessage({
```

Comment:
This can be up for discussion later, but instinctively, adding this text makes the whole trip summary feel too long.

Reply:
I'm open to alternatives! There has to be some way to indicate whether the accessibility summary is referring to the leg or the trip, I think this was the best idea we had without rewriting the gradation map or making a breaking change.

Reply:
But I agree with you, it is a little awkwardly phrased.

Reply:
Hm I think a way to make it more clear is "Wheelchair accessibility of this trip from `____` to `____` is `inaccessible/unknown`".

Reply:
I think the issue with that is that on any given list of trip results, they will all have the same origin and destination so it may feel a little redundant!